etc/emqx.conf

```diff
@@ -1651,6 +1651,12 @@ listener.ssl.external.reuseaddr = true
 ## Value: true | false
 ## listener.ssl.external.gc_after_handshake = false
 
+## hibernate the SSL process after idling for amount of time
+## Default: undefined (off)
+##
+## Value: Duration
+listener.ssl.external.hibernate_after = 5s
```

Comment:
Maybe keep commented to use the default (current behavior)?

Reply:
yes I think it is good idea